app/wizards/placements/add_hosting_interest_wizard.rb

```diff
@@ -121,6 +135,14 @@ def create_placements
           placement.save!
         end
       end
+
+      sen_quantity.times do
+        placement = Placement.create!(school:, academic_year: upcoming_academic_year, send_specific: true)
+        next if value_unknown(steps.fetch(:key_stage).key_stages)
+
+        placement.key_stages << selected_key_stages
+        placement.save!
```

Comment:
```suggestion
        placement = Placement.new(school:, academic_year: upcoming_academic_year, send_specific: true)
        next if value_unknown(steps.fetch(:key_stage).key_stages)

        placement.key_stages << selected_key_stages
        placement.save!
```

Would this be more efficient?

Reply:
No. Because a SEND placement can have NO key stages. The `next` escapes in this case. In your suggestion, the placement would not save.

Reply:
Ah yeah, fair enough! I should have considered that, my bad 👍